### PR TITLE
android/*: add German translation

### PR DESCRIPTION
**Pull request by marchersimon**
Our first fully translated platform.

### CODE REVIEW

File: pages.de/android/input.md
Comment (bl-ue) [inline pages.de/android/input.md:6]: There are 3 description lines here (excluding the link), but the [English page](https://github.com/tldr-pages/tldr/blame/4c059df5a66283a0a7c85d653af978b21342ea06/pages/android/input.md#L3-L6) only has two (besides the link).  Is this a mistake?

Also, I'm not sure but I don't think we're supposed to have more than two description lines (excluding the link, of course) 🤔

Comment (bl-ue) [reply pages.de/android/input.md:6]: Now I see that the English page is waiting for this change as well: https://github.com/tldr-pages/tldr/pull/5807/files#diff-6d7b5ac94fb71d5f0e18bbe601e0380866cd931813bd86356e2f3e722ae99fd4R5

**Review by sbrl — APPROVED** [review]
Wow thanks, @marchersimon!  Don't speak the language, but the syntax looks fine to me.

Thanks :slightly_smiling_face:

**Comment by sbrl** [comment]
We don't have any other people here who know German who could quickly cast their eye over this, do we?

**Comment by marchersimon** [comment]
There would be @FantasyCookie17, who recently contributed some German pages.

**Comment by sbrl** [comment]
Cool! Let's wait a day or so to see if they reply here - otherwise we can go ahead and merge tthis once we have 2 approvals.

**Comment by FantasyCookie17** [comment]
Thanks for notifying me, I'll look into it.

File: pages.de/android/dalvikvm.md
Comment (FantasyCookie17) [inline pages.de/android/dalvikvm.md:6]: In German, compound words are generally not separated by spaces. (pejoratively, this mistake is also referred to as the "Idioten Leerzeichen")
```suggestion
- Starte ein Java-Programm:
```

File: pages.de/android/dalvikvm.md
Comment (FantasyCookie17) [inline pages.de/android/dalvikvm.md:3]: Sounds off… German Wikipedia also refers to it as "Java Virtual Machine".
```suggestion
> Androids Java Virtual Machine.
```

File: pages.de/android/dumpsys.md
Comment (FantasyCookie17) [inline pages.de/android/dumpsys.md:3]:
```suggestion
> Stelle Informationen über Android-Systemservices bereit.
```

File: pages.de/android/am.md
Comment (FantasyCookie17) [inline pages.de/android/am.md:3]: Is this a proper name? If so, we should leave it at "activity manager". If not, we might want to avoid the "Idioten Leerzeichen".

File: pages.de/android/am.md
Comment (FantasyCookie17) [inline pages.de/android/am.md:18]:
```suggestion
- Konvertiere ein bestimmtes Ziel in einen URI:
```

File: pages.de/android/getprop.md
Comment (FantasyCookie17) [inline pages.de/android/getprop.md:3]:
```suggestion
> Zeige Informationen über Android-Systemeigenschaften.
```

File: pages.de/android/getprop.md
Comment (FantasyCookie17) [inline pages.de/android/getprop.md:6]:
```suggestion
- Gib Informationen über Anrdoid-Systemeigenschaften aus:
```

File: pages.de/android/getprop.md
Comment (FantasyCookie17) [inline pages.de/android/getprop.md:26]:
```suggestion
- Gib den OEM-Entsperrstatus aus:
```

File: pages.de/android/getprop.md
Comment (FantasyCookie17) [inline pages.de/android/getprop.md:30]:
```suggestion
- Gib die MAC-Adresse der WiFi-Karte aus:
```

File: pages.de/android/input.md
Comment (FantasyCookie17) [inline pages.de/android/input.md:3]:
```suggestion
> Sende Eventcodes oder Touchscreengesten an ein Android-Gerät.
```

File: pages.de/android/pm.md
Comment (FantasyCookie17) [inline pages.de/android/pm.md:14]:
```suggestion
- Gib eine liste aller installierten Drittpartei-Apps aus:
```

File: pages.de/android/pm.md
Comment (FantasyCookie17) [inline pages.de/android/pm.md:18]:
```suggestion
- Gib eine Liste aller Apps, auf die ein bestimmtes Schlüsselwort zutrifft, aus:
```

File: pages.de/android/settings.md
Comment (FantasyCookie17) [inline pages.de/android/settings.md:3]:
```suggestion
> Verwalte Android-Systemeinstellungen.
```

File: pages.de/android/settings.md
Comment (FantasyCookie17) [inline pages.de/android/settings.md:6]:
```suggestion
- Gib eine Liste aller Einstellungen im Namespace `global` aus:
```

File: pages.de/android/wm.md
Comment (FantasyCookie17) [inline pages.de/android/wm.md:3]:
```suggestion
> Zeige Informationen über den Bildschirm eines Android-Geräts.
```

File: pages.de/android/getprop.md
Comment (FantasyCookie17) [inline pages.de/android/getprop.md:14]:
```suggestion
- Gib das SDK-API-Level aus:
```

**Review by FantasyCookie17 — CHANGES_REQUESTED** [review]

File: pages.de/android/am.md
Comment (marchersimon) [reply pages.de/android/am.md:3]: I'd go with `Androids Aktivitäten-Manager` as you suggested before.

File: pages.de/android/pm.md
Comment (marchersimon) [reply pages.de/android/pm.md:14]: This sounds really strange. How about `Drittanbieter`?

File: pages.de/android/am.md
Comment (FantasyCookie17) [reply pages.de/android/am.md:3]: Alright. `Android-Aktivitäten-Manager` might also work, btw… 🤷🏻‍♀️

File: pages.de/android/pm.md
Comment (FantasyCookie17) [reply pages.de/android/pm.md:14]: Yeah, that sounds better. Upon consideration, 3rd-Party-Apps or Third-Party-Apps might also work; after all, there's nothing inherently wrong about anglicisms.

File: pages.de/android/getprop.md
Comment (FantasyCookie17) [inline pages.de/android/getprop.md:6]:
```suggestion
- Gib Informationen über Android-Systemeigenschaften aus:
```

**Review by FantasyCookie17 — CHANGES_REQUESTED** [review]
Just noticed a typo there.

**Comment by marchersimon** [comment]
Thanks @FantasyCookie17

**Review by bl-ue — APPROVED** [review]

**Review by FantasyCookie17 — APPROVED** [review]

**Comment by bl-ue** [comment]
>Thanks @FantasyCookie17 

Ja, danke! 😄

**Comment by FantasyCookie17** [comment]
Kein Problem, ich helfe gerne aus 😸  So, with three approvals in total, one by a native speaker, the other two by collaborators, would you like to merge this?

File: pages.de/android/logcat.md
Comment (FantasyCookie17) [inline pages.de/android/logcat.md:14]:
```suggestion
- Gib Zeilen aus, die einem regulären Ausdruck entsprechen:
```
Not necessarily necessary… Just a suggestion.

Comment (FantasyCookie17) [reply pages.de/android/logcat.md:14]: I also haven't looked into how other German pages do this, so 🤷🏻‍♀️

Comment (marchersimon) [reply pages.de/android/logcat.md:14]: I'm not entirely sure. In the english page we also use `regex` and not `regular expression`.

Comment (bl-ue) [reply pages.de/android/logcat.md:14]: I like the full name rather than the shortened jargon.  It sounds more professional.  Let's use it instead here and in the English page.

Comment (bl-ue) [reply pages.de/android/logcat.md:14]: You could also open a PR to standardize that.

Comment (bl-ue) [reply pages.de/android/logcat.md]: [`pages/common/httpflow.md`](https://github.com/tldr-pages/tldr/blob/521c8cd748736a1ef1da44e3650ff1e2dc30b333/pages/common/httpflow.md) uses `regexp`; that should be fixed too.

Comment (marchersimon) [reply pages.de/android/logcat.md]: Many pages use `{{regex}}` in their arguments, I assume that's fine. However, some use `regex` and some `regexp`. Should we normalize that as well?

**Comment by bl-ue** [comment]
>Kein Problem, ich helfe gerne aus 😸

Feel free to contribute in any way you like, as long as it's not too crazy 😆

>So, with three approvals in total, one by a native speaker, the other two by collaborators, would you like to merge this?

Definitely.  Let's git goin'! 🤠

File: pages.de/android/logcat.md
Comment (bl-ue) [reply pages.de/android/logcat.md]: Yeah I'd say normalize it all to `{{regular_expression}}`.  It it's too long in some places we can nip those occurences during the review.

**Pull request merged**
